### PR TITLE
Updating LAMMPS Sbatchfiles with new container; Correcting typo in Gr…

### PR DESCRIPTION
**Pull request by sanjtrip**
Updating LAMMPS files with new container; Correcting typo in Gromacs.

### CODE REVIEW

File: amdihapps/gromacs2gpu-mpi.sbatch [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 #SBATCH --nodes=1
-#SBATCH --ntasks-per-node=2
+#SBATCH --ntasks-per-node=4
```
Comment (amddcgpuce) [inline amdihapps/gromacs2gpu-mpi.sbatch:4]: @sanjtrip  WHy does 2gpu run needs 4 tasks per node?
@sanjtrip  please have your changes reviewed by @ozziemoreno @sidamd to catch typos and other issues.

@ozziemoreno @sidamd please comment here after your review

@sanjtrip have you tested these?

File: amdihapps/lammps1gpu.sbatch [hunk 2]
```diff
@@ -13,6 +13,6 @@ module load rocm/5.2.3
 tmp=/tmp/$USER/gpu1-$$
 mkdir -p $tmp
 
-singularity run --pwd /benchmark --writable-tmpfs /shared/apps/bin/lammps.sif /opt/lammps/tpl/openmpi/bin/mpirun -np 1 /opt/lammps/bin/lmp -k on g 1 -sf kk -pk kokkos cuda/aware on neigh full comm device neigh/qeq full newton on -v x 8 -v y 8 -v z 8 -v steps 600 -in in.lj -nocite -log /tmp/$USER/gpu1-$$/lj 
+singularity run --pwd /benchmark --writable-tmpfs /shared/apps/bin/lammps_2022.5.04_130.sif /opt/lammps/tpl/openmpi/bin/mpirun -np 1 /opt/lammps/bin/lmp -k on g 1 -sf kk -pk kokkos cuda/aware on neigh full comm device neigh/qeq full newton on -v x 8 -v y 8 -v z 8 -v steps 600 -in in.lj -nocite -log /tmp/$USER/gpu1-$$/lj
```
Comment (amddcgpuce) [inline amdihapps/lammps1gpu.sbatch:16]: @sanjtrip separate out issues into separate PRs. Do not bundle them.

File: amdihapps/gromacs2gpu-mpi.sbatch (hunk 1 above)
Comment (sanjtrip) [reply amdihapps/gromacs2gpu-mpi.sbatch:4]: @srinivamd, Discussed this with Roohollah. I had the same question for him. He said that Subin (?) gave him "-np 4" for mpi-version for 2-GPU config. ntasks-per-node has to match and that's why, I had to update it to 4 from usual setting of 2.

I have tested it.
